juriscraper/lib/string_utils.py

```diff
@@ -174,6 +177,26 @@ def titlecase(text, DEBUG=False):
 
     return text
 
+def _uppercase_word(word):
+    """
+    Helper function for uppercasing a word if it doesn't begin with Unicode characters.
+
+    This is needed due to differences between Python 2 and 3.
+    :param word: unicode string to uppercase
+    """
+    if six.PY2:
+        return CAPFIRST.sub(lambda m: m.group(0).upper(), word)
+    else:
+        try:
+            word[0].encode('ascii')
+            # doesn't start with unicode
+            return CAPFIRST.sub(lambda m: m.group(0).upper(), word)
+        except UnicodeEncodeError:
+            # starts with unicode
+            pass
```

Comment:
Seems like it's been this way a loooong time maybe since first commit: https://github.com/freelawproject/juriscraper/blob/626723b4373d65463e03d233bc1756128d039e1b/tests/tests.py